virttest/utils_misc.py

```diff
@@ -1562,6 +1562,54 @@ def add_identities_into_ssh_agent():
     process.run("ssh-add")
 
 
+def gen_lvmap_mpath(session=None):
+    """
+    Return the name mapping between logical volume and device
+    mapper multipathed device.
+
+    :param session: Guest session
+    :return: None or a dict like:
+            {"lv_name": dm_name}
+    """
+    names_map = None
+    cmd = ("lvdisplay|awk '/LV Name/{n=$3} "
+           "/Block device/{d=$3; sub(\".*:\", \"dm-\", d); print n,d;}'")
+
+    if session:
+        output = session.cmd_output(cmd).strip()
+    else:
+        output = process.system_output(cmd, shell=True)
+    lines = output.splitlines()
+    if len(lines) >= 1:
+        names_map = {}
+        for line in lines:
+            values = line.split()
+            lv_name, mpath_name = values[0], values[1]
+            names_map.update({lv_name: mpath_name})
+    return names_map
+
+
+def get_vg_mapped_blk_target(vg_name, session=None):
```

Comment:
parameter order is not consistent with your doc string

Reply:
fixed:)